_data/success-stories.yml

```diff
@@ -1,104 +1,108 @@
-    logo: cases-logos/waze.jpg
-    text: Thanks to Spinnaker, developers can focus on developing business logic rather than becoming experts on each cloud platform. Teams can track the lifecycle of a deployment using several notification mechanisms including email, Slack and SMS, allowing them to coordinate handoffs between developer and QA teams.
 
   - name: Avast
     link: Avast_10-25-19.pdf
     logo: cases-logos/avast.jpg
-    text: Spinnaker helped the team cut down the time and work needed for the versioning of ConfigMaps and secrets. That's a pain to do other ways. Maybe it seems simple, but it’s a big deal for our team. The whole goal here is to not wake people up in the middle of the night. The rollback and no-downtime deployment features are things everybody wants to do. The thing that Spinnaker promises is automated canary analysis. That to me is the holy grail to enabling continuous deployment.
+    text: The whole goal here is to not wake people up in the middle of the night. The rollback and no-downtime deployment features are things everybody wants to do. The thing that Spinnaker promises is automated canary analysis. That to me is the holy grail to enabling continuous deployment.
 
+  - name: Lookout
+    link: Lookout_10072019.pdf
+    logo: cases-logos/lookout.jpg
+    text: We knew our product had game changing potential in the government sector, but our existing deployment framework was too human-intensive and time consuming for FedRAMP compliance. Spinnaker provided the platform, support, and delivery pipeline we needed to bring our patch deployment times up to speed.	
+  
   - name: MakerBot
     link: Case%20Study%20Makerbot%20-%20Letter%20size_10-23-19.pdf
     logo: cases-logos/markerbot.jpg
     text: We originally envisioned transitioning toward Kubernetes on Google Cloud over three quarters. With Spinnaker’s open source flexibility, we completed the full migration in just 2.5 months. 
 
-  - name: Lookout
-    link: Lookout_10072019.pdf
-    logo: cases-logos/lookout.jpg
-    text: We knew our product had game changing potential in the government sector, but our existing deployment framework was too human-intensive and time consuming for FedRAMP compliance. Spinnaker provided the platform, support, and delivery pipeline we needed to bring our patch deployment times up to speed.	
+  - name: Netflix
+    link: https://medium.com/netflix-techblog/global-continuous-delivery-with-spinnaker-2a6896c23ba7
+    logo: cases-logos/netflix.jpg
+    text: The community uptake has been tremendous. Having a platform that is out-of-the-box accommodates multiple cloud providers and an extensible model to participate within, means that we are one large community rather than a bunch of micro-communities focused on various forks. This has the benefit that everyone within the community can take advantage of various innovations.
```

Comment:
```suggestion
    text: The community uptake has been tremendous. Having a platform that has an extensible model, and accommodates multiple cloud providers out of the box, means that we are one large community rather than a bunch of micro-communities focused on various forks. This has the benefit of allowing everyone within the community to take advantage of various innovations.
```

Reply:
Nice!

Reply:
Actually, GH doesn't let me commit your suggestion. 

![save](https://user-images.githubusercontent.com/29312809/69193289-6752d080-0af4-11ea-919a-9eb3990f2dc6.png)

Can you go ahead and commit the change? You might have unchecked the box that lets others contribute to your branch/PR. And if that's the case, it might also be impossible to merge Master in with you branch, if your branch gets out of date.

Reply:
![checkbox](https://user-images.githubusercontent.com/29312809/69194089-468b7a80-0af6-11ea-8d10-9489dae99615.png)